src/Ninja/G1CP/Content/Fixes/Gamesave/fix055_ReactivateInExtremo.d

```diff
@@ -0,0 +1,594 @@
+
+    // Now confirm the content of some functions
+    var int bytes[3];
+    var int matches;
+    var int count;
+    var int addr;
+    var int val;
+    var int good;
+
+    // Check that the content of "B_InsertInExtremo" is as expected
+    var int fncInsrtNpcId; fncInsrtNpcId = MEM_GetFuncID(Wld_InsertNpc);
+    addr = G1CP_GetFuncStart(fncInsertId);
+    good = TRUE;
+    repeat(i, GRIM); var int i; // All NPCs but Grim
+        if (MEM_ReadByte(addr) != zPAR_TOK_PUSHINT)        { good = FALSE; }; addr += 1;
+        if (MEM_ReadInt( addr) != MEM_ReadStatArr(npc, i)) { good = FALSE; }; addr += 4;
+        if (MEM_ReadByte(addr) != zPAR_TOK_PUSHVAR)        { good = FALSE; }; addr += 1+4;
+        if (MEM_ReadByte(addr) != zPAR_TOK_CALLEXTERN)     { good = FALSE; }; addr += 1;
+        if (MEM_ReadInt( addr) != fncInsrtNpcId)           { good = FALSE; }; addr += 4;
+    end;
+    if (MEM_ReadByte(addr) != zPAR_TOK_RET)                { good = FALSE; };
+    if (!good) {
+        MEM_Info("Content of function 'B_InsertInExtremo' not as expected");
+        return FALSE;
+    };
+
+    // Check that the content of "B_KillInExtremo" is as expected
+    var int fncExgRtnOff; fncExgRtnOff = G1CP_GetCallableOffsetI(fncExgRtnId);
+    addr = G1CP_GetFuncStart(fncRemoveId);
+    good = TRUE;
+    repeat(i, ALL); // All NPCs including Grim
+        if (MEM_ReadByte(addr) != zPAR_TOK_PUSHINT)        { good = FALSE; }; addr += 1;
+        if (MEM_ReadInt( addr) != MEM_ReadStatArr(npc, i)) { good = FALSE; }; addr += 4;
+        if (MEM_ReadByte(addr) != zPAR_TOK_PUSHVAR)        { good = FALSE; }; addr += 1+4;
+        if (MEM_ReadByte(addr) != zPAR_TOK_CALL)           { good = FALSE; }; addr += 1;
+        if (MEM_ReadInt( addr) != fncExgRtnOff)            { good = FALSE; }; addr += 4;
+    end;
+    if (MEM_ReadByte(addr) != zPAR_TOK_RET)                { good = FALSE; };
+    if (!good) {
+        MEM_Info("Content of function 'B_InsertInExtremo' not as expected");
+        return FALSE;
+    };
+
+    // Check that the content of "B_InExtremoStartMusic" is as expected
+    var int fncSndTrggrId; fncSndTrggrId = MEM_GetFuncID(Wld_SendTrigger);
+    addr = G1CP_GetFuncStart(fncStrtMsId);
+    good = TRUE;
+    if (MEM_ReadByte(addr) != zPAR_TOK_PUSHVAR)            { good = FALSE; }; addr += 5;
+    if (MEM_ReadByte(addr) != zPAR_TOK_CALLEXTERN)         { good = FALSE; }; addr += 1;
+    if (MEM_ReadInt( addr) != fncSndTrggrId)               { good = FALSE; }; addr += 4;
+    if (MEM_ReadByte(addr) != zPAR_TOK_PUSHVAR)            { good = FALSE; }; addr += 5;
+    if (MEM_ReadByte(addr) != zPAR_TOK_CALLEXTERN)         { good = FALSE; }; addr += 1;
+    if (MEM_ReadInt( addr) != fncSndTrggrId)               { good = FALSE; }; addr += 4;
+    val = MEM_ReadInt(addr+1);
+    if (MEM_ReadByte(addr) == zPAR_TOK_PUSHVAR) {
+        val = G1CP_GetIntI(val, 0, 0);
+    } else if (MEM_ReadByte(addr) != zPAR_TOK_PUSHINT)     { good = FALSE; }; addr += 1;
```

Comment:
Make these remaining checks optional (but ensure a final `zPAR_TOK_RETURN`). In that case, replace the second call to `Wld_SendTrigger` with some additional bytecode to set the variable:
```D
InExtremoPlaying = TRUE;
```